cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -101,6 +101,33 @@ export default {
   goToHoldingView: () => {
     cy.do(viewHoldingsButton.click());
     HoldingsRecordView.waitLoading();
+  },
+
+  checkHoldingsTable: (locationName, rowNumber, caption, barcode, status) => {
```

Comment:
check parameter caption usage

Reply:
It's not used since verification commented (because story not completed yet).